src/main/scala/com/campudus/tableaux/controller/MediaController.scala

```diff
@@ -209,12 +202,21 @@ class MediaController(override val config: TableauxConfig,
 
   def deleteFile(uuid: UUID): Future[TableauxFile] = {
     for {
+    // retrieve cells with this file for cache invalidation
+      cellsForFiles <- attachmentModel.retrieveCells(uuid)
```

Comment:
Shouldn't this call be after the deletion of the file? If the file is not found, we don't need to find all cells as there cannot be any of those...?